src/JokeraceEligibility.sol

```diff
@@ -135,39 +156,43 @@ contract JokeraceEligibility is HatsEligibilityModule {
     //////////////////////////////////////////////////////////////*/
 
   /**
-   * @notice Pulls the contest results from the jokerace contest contract.
+   * @notice Pulls the contest results from the next term's Jokerace contest contract and activates the next term.
    * @dev The eligible wearers for a given completed contest are the top K winners of the contest. In case there is a
    * tie, meaning that candidates in places K and K+1 have the same score, then the results of this contest are
    * rejected.
    */
-  function pullElectionResults() public returns (bool success) {
-    GovernorCountingSimple currentContest = GovernorCountingSimple(payable(underlyingContest));
+  function startNextTerm() public {
+    TermDetails memory nextTerm = terms[currentTermIndex + 1];
```

Comment:
<img width="693" alt="Screenshot 2024-08-27 at 14 42 30" src="https://github.com/user-attachments/assets/ec9f087c-3e1b-409e-8048-2c52c5a7b264">

All the details from the `nextTerm` object are emitted in the event at the end of the function. A different approach would be to just include the next term's index instead of all the details. That's a bit less good for whoever listens to the event because it will require a call to the contract in order to get the details of this new term. What do you think?